components/admin_console/admin_sidebar/admin_sidebar.tsx

```diff
@@ -333,6 +311,7 @@ class AdminSidebar extends React.PureComponent<Props, State> {
                     />
                 </div>
                 <Scrollbars
+                    // eslint-disable-next-line react/no-string-refs
                     ref='scrollbar'
```

Comment:
This can actually be removed rather than moving the comment. It doesn't look like it's actually used

Reply:
removed it